sdk/ai/azure-ai-projects/tests/test_base.py

```diff
@@ -101,7 +102,7 @@ class TestBase(AzureRecordedTestCase):
     }
 
     test_agents_params = {
-        "model_deployment_name": "gpt-4o",
+        "model_deployment_name": os.environ.get("AZURE_AI_PROJECTS_TESTS_AGENTS_MODEL_DEPLOYMENT_NAME") or "gpt-4o",
```

Comment:
@dargilco I am wondering whether there is some extra work with the service preparer that I will need to do in this area once we have working recordings for these tests? Since we dont have working recordings today it was hard for me to be sure.